ui/app/pages/create-account/connect-hardware/index.scss

```diff
@@ -132,6 +113,14 @@
     color: #9b9b9b;
     margin-top: 10px;
     margin-bottom: 15px;
+
+    &__link {
```

Comment:
I would really appreciate if we tried to stick to BEM-compatible classnames, which are of the format "[block]__element--modifier". This classname how has two `__` delimiters, which isn't valid BEM. Instead I'd suggest `-link` here instead of `__link`.

It's a pervasive pre-existing pattern I know,  but I've been trying not to let it spread further.

Reply:
Updated!